File: schemas/substitute/substitute_schema.py
from typing import List

from pydantic import BaseModel

from schemas.substitute.attribute_schema import Attribute
from schemas.substitute.modification_schema import Modification

class SubstituteSchema(BaseModel):
    Type: str
    Name: str
    ModelId: int
    Modification: Modification
    Attributes: List[Attribute]

File: OzonDomains/TecDocModels/Substitute/SubstituteSchema.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OzonDomains.TecDocModels.Substitute
{
    public class SubstituteSchema
    {
        public string Type { get; set; }
        public string Name { get; set; }
        public Modification Modification { get; set; }
        public List<Attribute> Attributes { get; set; }
    }
}
